### PR TITLE
#20741 - Add foreman_scc_manager

### PR DESCRIPTION
**Pull request by Klaas-**
from https://github.com/ATIX-AG/foreman_scc_manager
Foreman plugin to sync SUSE Customer Center products and repositories into Katello
as mentioned in the community demo: https://youtu.be/uUFsHWF292M?t=2372

### CODE REVIEW

**Comment by Klaas-** [comment]
@ehelms do I need a redmine issue for this?

**Comment by Klaas-** [comment]
I had to add foreman-tasks restart to posttrans for this plugin because it uses foreman-tasks

**Comment by ehelms** [comment]
[test]

File: rubygem-foreman_scc_manager/rubygem-foreman_scc_manager.spec [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+%files doc
+%doc %{gem_docdir}
+%doc %{gem_instdir}/README.md
+%{gem_instdir}/Rakefile
+%{gem_instdir}/test
+
+%posttrans
+%{foreman_db_migrate}
+%{foreman_restart}
+/usr/bin/systemctl restart foreman-tasks
```
Comment (ehelms) [inline rubygem-foreman_scc_manager/rubygem-foreman_scc_manager.spec:83]: Hopefully this doesn't blow anything up

File: rubygem-foreman_scc_manager/rubygem-foreman_scc_manager.spec [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+# Create the gem as gem install only works on a gem file
+%{?scl:scl enable %{scl} - << \EOF}
+gem build %{gem_name}.gemspec
+%{?scl:EOF}
+
+# %%gem_install compiles any C extensions and installs the gem into ./%%gem_dir
+# by default, so that we can move it into the buildroot in %%install
+%{?scl:scl enable %{scl} - << \EOF}
+%gem_install
+%{?scl:EOF}
```
Comment (ehelms) [inline rubygem-foreman_scc_manager/rubygem-foreman_scc_manager.spec]: Unless this gem has bin files or c-extensions, you can reduce this down to using just these three lines in the prep phase similar to rubygem-katello

**Comment by Klaas-** [comment]
thanks @ehelms

**Pull request merged**